src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall(){
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
+        Random random = new Random(100);
+        ball.setNumber(random.nextInt());
```

Comment:
The `random.nextInt()` method should be called with an argument specifying the upper bound. Use `random.nextInt(100)` to ensure the random number is within the desired range.